src/Screens/HomeScreen/HomeScreen.tsx

```diff
@@ -8,26 +8,42 @@ import { useTranslation } from "react-i18next";
 import { fetchTodos } from "store/todo/todoActions";
 import { useDispatch } from "react-redux";
 import Form from "components/Form/Form";
+import { StyleSheet } from "aphrodite";
+import CustomButton from "components/CustomButton/CustomButton";
 
 const style = bemCssModule(HomeScreenStyles);
 
 const HomeScreen = () => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
-  const btnAddTask = <button>Add task</button>;
```

Comment:
We still want to pass our new button component in actionButton prop :) Just make const button = <CustomButton .... />
and pass it into actionButton property .